recipe/conda_build_config.yaml

```diff
@@ -37,9 +32,11 @@ conda_glibc_ver:
 ctng_vendor:
   - conda
 
+clang_version:
+  - 16
```

Comment:
I'm asking because `clang_linux-*` in this feedstock means we cannot easily build for several clang-versions, which is how we've done it for https://github.com/conda-forge/clang-win-activation-feedstock, and would IMO be good to have for linux as well.

In practice we might build up to three clang versions (same for all clang activation feedstocks):
* the version in the global pinning (mostly `v${latest - 1}`)
* the last clang release (`v${latest}`)
* the pre-release of the next version (`v${latest + 1}.rc`)

Occasionally we also build the version before the current pinning (e.g. currently 15), to keep the compilers current for feedstocks who couldn't do the last compiler migration (there are a few feedstocks that needed to pin <16 due to clang becoming stricter about some thing). This usually gets dropped at the lastest when the first rc's of the next versions are built, so we don't exceed the stated "up to three versions" there.

Of course, one can argue that this is not essential - but it would be nice to be able to use consistent clang versions also for (testing) the newest versions or rc's.